src/server/robots/useCases/générerRobots.useCase.ts

```diff
@@ -0,0 +1,16 @@
+import { CRAWLER_NOT_PRODUCTION_MODE, CRAWLER_PRODUCTION_MODE } from '../domain/robots';
+
+const PRODUCTION_ENVIRONMENT = 'production';
+
+export class GénérerRobotsUseCase {
+	constructor(private environment: string) {
+	}
+
+	async handle(): Promise<string> {
```

Comment:
async handle(): Promise<string> {
	const isProduction = this.environment === PRODUCTION_ENVIRONMENT;
		const crawlerMode = isProduction ? CRAWLER_PRODUCTION_MODE : CRAWLER_NOT_PRODUCTION_MODE;
		return Promise.resolve(crawlerMode);
	}